python/paddle/fluid/tests/unittests/op_test.py

```diff
@@ -503,6 +530,13 @@ def check_output(self,
                                          check_dygraph)
 
     def check_output_customized(self, checker):
+        self._check_output_customized(checker)
+        if self._is_gc_test_enabled():
+            self._enable_gc(True)
+            self._check_output_customized(checker)
+            self._enable_gc(False)
```

Comment:
Same above.

Reply:
For safety check, though I have not figured out any case that the calculating result would be different if using gc or not.